tests/framework/matcher/status.go

```diff
@@ -82,3 +84,26 @@ func HaveStateChangeRequests() gomegatypes.GomegaMatcher {
 		}),
 	}))
 }
+
+func InterfaceByName(interfaceName string) func(vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
no need to export this function, no one is using it outside this file.

Reply:
It make sense to make this public, it's something that other developers may want to use.

Reply:
ok, but, I would say let the first person that needs to have this public make it public.